cltk/corpus/odia/alphabet.py

```diff
@@ -0,0 +1,26 @@
+'''Odia is an Indian language, belonging to the Indo-Aryan branch of
```

Comment:
It's better to add this description to the docs.
See cltk docs style https://github.com/cltk/cltk/blob/master/docs/greek.rst

Reply:
Okay . I'll make the changes .